test/Numerics/Mesh/mpi_connect.jl

```diff
@@ -68,7 +68,7 @@ function main()
         nrealelem = 5
         globalelems = [1, 2, 3, 4, 5, 6, 7, 8, 14, 15]
         elemtoelem_expect = [
-            1 1 4 4 5 6 7 8 9 10
+            1 1 4 2 3 4 7 8 9 10
```

Comment:
I don't understand why these numbers changed

Reply:
`elemtoelem` now stores boundary element number if it faces a boundary (previously it just pointed back to the element itself). In this case, face 1 of elements [1,4,5,6] face boundary 1, and so are numbered `[1,2,3,4]`

Reply:
I see that now. Thanks. (Some how missed that line 🤦 )